### DISCUSSION TITLE
Multiphase Multicomponent Microstructure Simulation

### DISCUSSION THREAD

**Discussion by thestormthunder** [idaholab/moose#27542] [Q&A Modules: Phase field]
Hello Moose Community,

I hope you all are doing great.  I am here with a request to help me simulate the microstructure of a Multicomponent Multiphase alloy system using Moose at different heat treatment conditions. I am very new to phase field simulation and also to Moose. Request you to kindly help me in this regard.

I have experience in CALPHAD simulation.

Thanks for your time.

**Comment by GiudGiud** [comment]
@amjokisaari @laagesen

**Comment by laagesen** [comment]
Hello and thanks for your interest in the MOOSE framework! Where are you located? If you are able to attend the upcoming MOOSE training workshop this summer, that would help you get up to speed quickly:

https://mooseframework.inl.gov/training/index.html
